lib/android/app/src/main/java/com/reactnativenavigation/views/stack/topbar/titlebar/MainToolBar.kt

```diff
@@ -15,15 +15,25 @@ import com.reactnativenavigation.utils.CompatUtils
 import com.reactnativenavigation.utils.UiUtils
 import com.reactnativenavigation.utils.ViewUtils
 import com.reactnativenavigation.utils.logd
+import kotlin.math.max
+import kotlin.math.min
 
 const val DEFAULT_LEFT_MARGIN = 16
+const val SYSTEM_ACTION_BUTTON_SIZE_DP = 48
 
 class MainToolBar(context: Context) : RelativeLayout(context) {
```

Comment:
Since `MainToolBar` extends `RelativeLayout`, not `Toolbar` - I'd rename the class to something else. It's basically a `TitleAndButtonsContainer`. Its purpose is to layout and measure the title and buttons.